src/models/PurchasedAlbum.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const PurchasedAlbumSchema = new mongoose.Schema({
+  albumId: { type: Schema.Types.ObjectId, ref: 'Album', required: true },
+  userId: { type: Schema.Types.ObjectId, ref: 'User', required: true },
+});
+
+const PurchasedAlbum = mongoose.model('AlbumPurchased', PurchasedAlbumSchema);
```

Comment:
```suggestion
const PurchasedAlbum = mongoose.model('PurchasedAlbum', PurchasedAlbumSchema);
```
(since we're referring to the model as PurchasedAlbum elsewhere, like in the virtuals, it has to match this string.

Reply:
Otherwise we get the error `Uncaught MissingSchemaError: Schema hasn't been registered for model "PurchasedAlbum".` when we try to use the virtuals, because the virtuals code will look for `PurchasedAlbum` but here we've defined the model to have the name `AlbumPurchased`.